accessibilite.md

```diff
@@ -0,0 +1,132 @@
+# Accessibilité 
+
+Initialement relatif au monde du handicap, des enfants ou des personnes âgées, l’accessibilité c’est étendu à l'ensemble des citoyens et utilisé pour désigner l'accès à de nombreux domaines : **physique**, la liberté de déplacement dans l’espace (mobilité) ; **éducatif**, le droit à une scolarisation ; **civique**, le droit de vote ; **culturel**, pouvoir développer sa culture ; **numérique**, adaptation des systèmes numériques, dont les sites web, aux différents types de handicap, développement d'outils spécifiques tels loupe ou clavier visuel...
+
+
+[[toc]]
+
+## Comprendre le RGAA 4 
```

Comment:
Tu ne dis pas ce que signifie cet acronyme.

Tu peux définir un acronyme au début du document de cette façon :
```
*[DoS]: Denial of Service
```

Reply:
Merci c'est noté 👍